packages/opentelemetry-metrics/src/export/Controller.ts

```diff
@@ -44,13 +44,10 @@ export class PushController extends Controller {
 
   private async _collect() {
     await this._meter.collect();
-    await this._exporter.export(
-      this._meter.getBatcher().checkPointSet(),
-      result => {
-        if (result !== ExportResult.SUCCESS) {
-          // @todo: log error
-        }
+    this._exporter.export(this._meter.getBatcher().checkPointSet(), result => {
```

Comment:
```suggestion
    return new Promise((resolve, reject) => {
      this._exporter.export(
        this._meter.getBatcher().checkPointSet(),
        result => {
          if (result === ExportResult.SUCCESS) {
            resolve();
          } else {
            reject();
          }
        }
      );
    });
```

Reply:
@obecny I implemented this while also updating the return type and leaving in the @todo comment, I think there is still error logging functionality that needs to be handled so we should likely leave that in, no?